app/i18n/en/index.ts

```diff
@@ -728,6 +728,7 @@ const en: BaseTranslation = {
   PhoneValidationScreen: {
     errorLoggingIn: "Error logging in. Did you use the right code?",
     errorTooManyAttempts: "Too many attempts. Please try again later.",
+    errorCannotUpgradeToExistingAccount: "This phone account already exists. Please log out of your trial account and then log in with your phone number.",
```

Comment:
have we settled down on the `trial` account terminology?

Reply:
Right now yes.